common/config/rush/pnpm-lock.yaml

```diff
@@ -270,9 +273,9 @@ importers:
       '@itwin/core-quantity': ^3.2.0
       '@itwin/core-react': ^3.2.0
       '@itwin/eslint-plugin': ^3.2.0
-      '@itwin/imodel-browser-react': ^0.12.1
+      '@itwin/imodel-browser-react': ^0.12.3
       '@itwin/imodel-components-react': ^3.2.0
-      '@itwin/insights-client': ^0.3.0
+      '@itwin/insights-client': 0.2.1
```

Comment:
Don't downgrade the insights-client.

Reply:
Forgot to run rush update after merge. Done.